lib/rspec/core/ordering.rb

```diff
@@ -78,6 +86,7 @@ def initialize(configuration)
           @strategies    = {}
 
           register(:random,  Random.new(configuration))
+          register(:modification_time,  ModificationTime.new)
```

Comment:
Nitpick, cosmetic: I believe the double space is here to align the indentation, but it's not necessary anymore.